libs/xc/xenctrl.ml

```diff
@@ -310,6 +310,12 @@ external get_featureset : handle -> featureset_index -> int64 array = "mock1"
 external upgrade_oldstyle_featuremask: handle -> int64 array -> bool -> int64 array = "mock1"
 external oldstyle_featuremask: handle -> int64 array = "mock1"
 
+(* New MSR external functions *)
+external xc_get_msr_arch_caps : handle -> int64 = "mock2"
```

Comment:
Sorry to bring it up again - but this function has 1 parameter (of type `handle`), hence `"mock1"` would be correct.